vembrane/modules/filter.py

```diff
@@ -184,7 +194,17 @@ def statistics(
         yaml.dump(dict(counter), out)
 
 
+def read_auxiliary(aux) -> Dict[str, Set[str]]:
+    # read auxiliary files, split at any whitespace and store contents in a set
+    def read_set(path: str) -> Set[str]:
+        with open(path, "rt") as f:
+            return set(f.read().split())
```

Comment:
This seems dangerous to me. I think I'd rather expect each entry to be in one line. This way, there can be whitespace in the entries.

Reply:
```suggestion
            return set(l.strip() for l in f.readlines())
```

Reply:
```suggestion
            return set(l.rstrip() for l in f)
```